src/iexamine.cpp

```diff
@@ -974,23 +974,35 @@ void iexamine::slot_machine(player *p, map*, const tripoint&)
 void iexamine::safe(player *p, map *m, const tripoint &examp)
 {
     if (!p->has_amount("stethoscope", 1)) {
-        add_msg(m_info, _("You need a stethoscope for safecracking."));
-        return;
+        if (query_yn(_("The safe is locked. Input a random combination?"))) {
+            p->moves -= 100;
+            // one_in(30^3) chance of guessing
+            if (one_in(27000)) {
```

Comment:
It doesn't prompt at all if the player can actually crack the safe, precisely to avoid adding another query to the menu. It's more just there for giggles than anything else, because honestly in real life there's no way that you are going to be opening a safe by entering random combinations; with most real safes even if we assumed you could enter in 1 whole combination every 3 seconds it would still take you around 1.5 years before you even hit the 50/50 chance of having hit the right combination.